conf/modules.config

```diff
@@ -87,52 +87,52 @@ params {
             args2            = 'sort'
             publish_files    = false
         }
-        'bwa_mem2_mem' {
+        'bwa_mem1_mem' {
             args             = '-K 100000000 -M'
             args2            = 'sort'
             publish_files    = false
         }
-        'bwa_mem2_mem_tumor' {
-            args             = '-K 100000000 -M -B 3'
-            args2            = 'sort'
-            publish_files    = false
-        }
         'samtools_index_mapping' {
             publish_by_meta  = true
             publish_files    = ['bai':'mapped']
             publish_dir      = 'preprocessing'
         }
-        // MARKDUPLICATES
-        'markduplicates' {
-            args             = 'REMOVE_DUPLICATES=false VALIDATION_STRINGENCY=LENIENT'
-            suffix           = '.md'
+        'merge_bam_mapping' {
             publish_by_meta  = true
+            publish_files    = ['bam':'mapped']
             publish_dir      = 'preprocessing'
+        }
+        'seqkit_split2' {
+            args             = "--by-size ${params.split_fastq}"
             publish_files    = false
         }
-        'markduplicatesspark' {
-            args             = '--remove-sequencing-duplicates false -VS LENIENT'
+        // MARKDUPLICATES
+        'estimatelibrarycomplexity' {
+            args             = ''
             suffix           = '.md'
             publish_by_meta  = true
             publish_dir      = 'preprocessing'
-            publish_files    = ['cram': 'markduplicates', 'crai': 'markduplicates']
+            publish_files    = ['metrics': 'markduplicates']
         }
-        'estimatelibrarycomplexity' {
-            args             = ''
+        'markduplicates' {
+            args             = 'REMOVE_DUPLICATES=false VALIDATION_STRINGENCY=LENIENT'
             suffix           = '.md'
             publish_by_meta  = true
             publish_dir      = 'preprocessing'
-            publish_files    = ['metrics': 'markduplicates']
```

Comment:
shouldn't e publish the files same as with spark, or the other way around for consistency?

Reply:
No, because if we don't use spark, then markduplicates makes BAMs, and we only keep the CRAMs out of `SAMTOOLS_BAM_TO_CRAM`

Reply:
🤦‍♀️ right.... 😆